src/index.js

```diff
@@ -17,7 +17,14 @@ export const defaultBreakpoints = {
   small: '450px',
 };
 
-function getSizeFromBreakpoint(breakpointValue, breakpoints = {}) {
+/**
+ * Maps size to breakpoints array
+ *
+ * @param {*} breakpointValue - The value to map
+ * @param {Object} breakpoints - Map labels to breakpoint sizes
+ * @return {string|*}
+ */
+function getSizeFromBreakpoint(breakpointValue, breakpoints) {
```

Comment:
Also here, why the default breakpoints was been removed?

Reply:
Same as above. We don't have to fear `breakpoints` could ever be empty, as `generateMedia` gets called with `(breakpoints = defaultBreakpoints)` when no arguments are given. Also, with them set 100% coverage would never have been possible, as their branches were never taken cause of this ; ).